.github/ISSUE_TEMPLATE/71_org_editor-recruit_reviewers.md

```diff
@@ -0,0 +1,28 @@
+---
+name: "[Organizer -> Editor] Recruit Reviewers"
+about: Checklist for Associate Editor to fill out prior to review
+title: "[PRE-REVIEW] Recruit Reviewers"
+labels: pre-review
+assignees: ''
+---
+
+<!--
+## DO NOT EDIT THIS FILE OUTSIDE OF THE journalovi/jovi-workflows REPOSITORY
+##
+## This file is automatically updated in all repositories within the journalovi
+## Github organization whenever the version in journalovi/jovi-workflows is
+## changed, so any other edits will be overwritten. To update this file, make
+## a commit or pull request at https://github.com/journalovi/jovi-workflows
+-->
+
+### Instructions for Editor
+- Send out review requests using the [reviewer invitation email template](https://docs.google.com/document/d/15ttXvQg7g5JEk2Yzfd9eXSN6oIpc_A61qbuwygthuG0/edit?usp=sharing).
+- When a reviewer agrees to review:
+  1. Create an issue using the *Review Request* Issue template and assign it to the reviewer.
```

Comment:
Can we have the link to the correct template here? (Something like `new?labels=…` that you made in  `20_editor_reviewer-write_review.md`.

Reply:
Yeah, I plan to update all similar references to be links before merging.

Reply:
done